src/test/java/core/basesyntax/serviceimpltest/ReaderServiceImplTest.java

```diff
@@ -0,0 +1,66 @@
+package core.basesyntax.serviceimpltest;
```

Comment:
the package should be named as well as the tested class's package is named
e.g.
`core.basesyntax.service`

check the other package names